introduction.md

```diff
@@ -1,3 +1,8 @@
 My name is Anders H.Abrahamsen and I am a senior developer situated in Bergen, Norway. I have worked with Git for 4 years and found that it was time to go through some courses to make sure I know about every little detail that I might have missed during daily use.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?